app/import_types.go

```diff
@@ -209,11 +215,45 @@ type LineImportWorkerError struct {
 }
 
 type AttachmentImportData struct {
-	Path *string   `json:"path"`
-	Data *zip.File `json:"-"`
+	Id            string
+	CreatorId     string
+	PostId        string
+	ChannelId     string
+	CreateAt      int64
+	UpdateAt      int64
+	Path          *string
+	ThumbnailPath string
+	PreviewPath   string
+	Name          string
+	Extension     string
+	Size          int64
+	MimeType      string
+	Width         int
+	Height        int
+	Data          *zip.File
 }
-
 type ComparablePreference struct {
 	Category string
 	Name     string
 }
+
+type LogFormat struct {
+	Id            string
+	CreatorId     string
+	PostId        string
+	ChannelId     string
+	CreateAt      time.Time
+	UpdateAt      time.Time
+	Path          string
+	ThumbnailPath string
+	PreviewPath   string
+	Name          string
+	Extension     string
+	Size          int64
+	MimeType      string
+	Width         int
+	Height        int
+}
+type MissingAttachments struct {
```

Comment:
why create this struct explicitly check the prefix option in marshalIndent function and see if you can use it here

Reply:
prefix option in marshalIndent function append before each line